Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Recurring_rare_domain_access_from_an_unsigned_process.yml

```diff
@@ -0,0 +1,951 @@
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: f30c9dc6-ffdd-4756-8b06-4499c0f4e588
+    type: regular
+    task:
+      id: f30c9dc6-ffdd-4756-8b06-4499c0f4e588
+      version: -1
+      name: CommandLine Analysis
+      description: |-
+        This script evaluates command-line threats by analyzing both original and decoded inputs. It assigns weighted scores to detected patterns, such as AMSI bypass or credential dumping, and applies risk combination bonuses for multiple detections. The total score is normalized to a 0-100 scale, with risk levels categorized as follows:
+
+        * 0-25: Low Risk
+        * 26-50: Medium Risk
+        * 51-90: High Risk
+        * 91-100: Critical Risk
+
+        The scoring mechanism provides a comprehensive risk assessment, considering both the severity and frequency of malicious behaviors.
+      scriptName: CommandLineAnalysis
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "28"
+    scriptarguments:
+      command_line:
+        simple: ${alert.initiatorcmd}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 58.5,
+          "y": 522
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: cb1c082e-5e12-42a1-8a27-25184f8602f8
+    type: title
+    task:
+      id: cb1c082e-5e12-42a1-8a27-25184f8602f8
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "9"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 62,
+          "y": 999
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 02333818-0b33-4334-815c-f57124954135
+    type: title
+    task:
+      id: 02333818-0b33-4334-815c-f57124954135
+      version: -1
+      name: Inconclusive
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "18"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 545.5,
+          "y": 999
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: d17b132c-b5a2-4497-83aa-8dc22a290f6b
+    type: regular
+    task:
+      id: d17b132c-b5a2-4497-83aa-8dc22a290f6b
+      version: -1
+      name: Terminate Causality - Action Process
```

Comment:
You specify "Terminate Causality" with "Action Process"
Use "core-terminate-causality"
And use as input "actorprocessinstanceid"
It's unclear which process we want to target here.